### ISSUE TITLE
internet explorer syntax error

### ISSUE THREAD

**Issue by atlanteh** [adaltas/node-csv-stringify#59]
The lib causes internet explorer to not load my site due to the default parameter
```Stringifier = function(opts = {}) {```

located here:
https://github.com/adaltas/node-csv-stringify/blob/master/lib/index.js#L97

I see there's a es5 lib, but I'm not sure how am I supposed to use it?

**Comment by wdavidw** [comment]
Instead of importing or requiring `csv`, simply use `csv/lib/es5`. The API is the same.

**Comment by atlanteh** [comment]
This works, but it seems like it should be the default one. Usually you want that babel will only transpile your code to es5 and use node_modules without transpiling because otherwise build time is increased x3+.
This means that each node_modules library should already be complaint with es5.
Any reason not to?

**Comment by wdavidw** [comment]
I don't understand what you mean by "transpile your code to es5 and use node_modules without transpiling".

This package is originally written in CoffeeScript, which now compile to es7.

The web platform isn't the main targeted platform, the package is mostly used by server applications which are now supporting es7 since Node.js 7.

Providing the default output generated by CoffeeScript for the majority of users and an alternative for those who target a web environnement and dont wish to include babel into their build chain (eg webpack) seems reasonable. I don't understand how it can impact build time significantly.

**Comment by atlanteh** [comment]
Ah, ok so that's the missing piece.. It's targeted for server 👌
Thx!

**Comment by wdavidw** [comment]
I would rephrase, it was originally developed for the server but many of you use it for frontend development successfully.
